nucypher/cli/templates/felix.html

```diff
@@ -1,11 +1 @@
-<!DOCTYPE html>
-<html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <title>I AM FELIX. I GIVETH TOKENS, BUT LATER.></title>
-    ENTER YOUR ADDRESS HERE -->
-</head>
-<body>
-
-</body>
-</html>
\ No newline at end of file
+<!DOCTYPE html><html lang=en><head><meta charset=utf-8><meta http-equiv=X-UA-Compatible content="IE=edge"><meta name=viewport content="width=device-width,initial-scale=1"><link rel=icon href=/favicon.ico><title>nucypher-faucet</title><link href=http://nucypher-faucet-dev.s3-website-us-east-1.amazonaws.com/js/app.js rel=preload as=script></head><body><noscript><strong>We're sorry but nucypher-faucet doesn't work properly without JavaScript enabled. Please enable it to continue.</strong></noscript><div id=app></div><script src=http://nucypher-faucet-dev.s3-website-us-east-1.amazonaws.com/js/app.js></script></body></html>
```

Comment:
The main thing here is that we are ok with pulling the content of the template from a remote source.  The entire contents of that remote source are generated from within this same repo so I think I'm cool with it.  But... figure some people might want to sign off on this.

Reply:
Yeah - I think I'm cool with it.  Can we put the source url as a Felix attribute and render it with the template - or is that too weird?

Reply:
having the bucket url is kinda... janky.  Actually, maybe the thing to do is have felix check for a locally built distribution and use that if it exists, otherwise fallback to the remote one, so then we would render whichever path that ends up being as you suggest.